perftests/standalone-enc/Makefile.BSD

```diff
@@ -14,51 +14,15 @@ PROTO_DIR	=	../../proto
 # Main test code
 SRCS	=	main.c
 
-# spipe protocol
+# spiped protocol
```

Comment:
This probably deserves an extra comment pointing out that this isn't a mistake and we're deliberately recompiling because we want different compiler options from what was used in liball.a.

Reply:
Changed it to:
```
# spiped protocol, with an extra -DSTANDALONE_ENC_TESTING to enable mkkeypair()
```